src/modules/Shadow/info.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "shadow",
+  "description": "Creates a two sided shadow.",
+  "inputs": {},
+  "docs-link":"https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#shadow-module"
```

Comment:
This doesn't exist, Could you write the docs also?

Reply:
Ill open a new issue for this. Any newcomer can work on documentation part as a good first time issue. @blurry-x-face